app/code/Apruve/Payment/Model/PaymentMethod.php

```diff
@@ -7,8 +7,10 @@ class PaymentMethod extends \Magento\Payment\Model\Method\AbstractMethod
     const CODE = 'apruve';
     const DISCOUNT = 'Discount';
 
+    const AUTHORIZE_ACTION = 'finalize';
+    const OFFLINE_ACTION = '';
```

Comment:
Why did this become blank?

Reply:
I made that OFFLINE_ACTION blank because that is what is required to make the _apruve method call work for offline orders. I just wanted to make this more explicit.